frontend/packages/dev-console/integration-tests/features/customization/customization-of-web-terminal.feature

```diff
@@ -0,0 +1,83 @@
+@customize-web-terminal @odc-7283
+Feature: Customization of web terminal options
+                As admin you can change the default timeout and image of Web Terminal.
+
+        Background:
+            Given user has installed Web Terminal operator
+              And user is at developer perspective
+              And user has created or selected namespace "aut-web-terminal"
+
+        @smoke
+        Scenario: When navigates to cluster configuration page: WT-02-TC01
+            Given user is at Consoles page
+             When user navigates to Cluster configuration page
+              And user clicks on Web Terminal tab
+             Then user should see Web Terminal Congiguration page
+
+
+        @regression
```

Comment:
I think this is not a regression as it is a new feature. @sanketpathak could maybe help even more with the right labels and e2e tests.